students/alex_whitty/lesson4/trigrams.py

```diff
@@ -0,0 +1,83 @@
+import random
+import string
+#!/usr/bin/env python3
+def parse_file():
+    '''
+    Takes a hardcoded file name and returns a list of words
+    '''
+    # Setup a translator to remove punctuation
+    translator = str.maketrans('', '', string.punctuation)
+    # Setup an empty list to contain the full list of words in the text
+    full_list = []
+    with open('double_take.txt', 'r') as f:
+        for line in f:
+            # This will take each line, remove punctuation and then split it
+            # into a list of words and attach this list to the full_list
+            # note the use of "extend()" instead of "append()"
+            # TODO: Figure out how to remove carriage returns
+            full_list.extend(line.translate(translator).split(' '))
+    return full_list
+
+
+
+
+trigrams = {("When", "the"): ['Travelers'],
+            ("the", "Travelers"): ["from"],
+            ("Travelers", "from"): ["Outer"],
+            ("from", "Outer"): ["space"],
+            ("Outer", "Space"): ["dug"],
+            ("Space", "dug"): ["into"],
+            ("dug", "into"): ["the"],
+            ("into", "the"): ["pile"],
+            }
```

Comment:
Remove this dictionary to avoid confusion, as the code is already capable of generating it's own trigram dict.